src/clj/chronograph/middleware.clj

```diff
@@ -1,6 +1,8 @@
 (ns chronograph.middleware
-  (:require [taoensso.timbre :as log]
-            [ring.util.response :as response]))
+  (:require [chronograph.auth :as auth]
+            [chronograph.db.user :as users-db]
```

Comment:
Let's use the function from `domain.user` instead of `db.user`.

Reply:
eeek. fixed.